hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -157,6 +158,7 @@ public void handle(SCMCommand command, OzoneContainer container,
       };
       updateCommandStatus(cmd.getContext(), cmd.getCmd(), updateFailure, LOG);
       LOG.warn("Command is discarded because of the command queue is full");
+      blockDeleteMetrics.incrTotalCommandsDiscarded(1);
```

Comment:
chances of this occurence is rare now as per SCM track queue size of dn also. so there will not be rejection / will be very rare. This metrics will not be useful.

Reply:
Yes this can be ignored as chances is rare now this to happen, due to recent improvements.